### DISCUSSION TITLE
Audio bug

### DISCUSSION THREAD

**Discussion by Freshprince24** [ikemen-engine/Ikemen-GO#1643] [Q&A]
Okay so I've made a character for mugen 1.1. Released it and received some complaints that it crashes the nightly build as well as not having functional audio in other versions of Ikemen. Know it's not my snd file because I tried it in another character and that played out fine. Could hear the sounds. Assumed it was my code and then tried the same triggers as the other character for my Playsnd state controllers. Still no luck in the actual character. 
I'll leave a link to the character if anyone can take a look at it and get back to me I'd greatly appreciate. Tried the discord and was sent here. 

Character link
https://www.mediafire.com/file/p4t7iamb6j6fetq/Android_21.rar/file 

If that's suspicious heres the release video with a couple comments having the issue

https://youtu.be/MwbDOTCOnxg?si=20_jF1ZXsB328DyO

Again. Works fine in mugen 1.1, is silent in regular Ikemen, and crashes on nightly. None of my previous characters have this issue and I havent really done anything new in how I handle sound. If someone can help that would be appreciated. Saw someone with a similar problem here but they never replied when asked to submit the character

**Comment by Eiton** [comment]
![image](https://github.com/ikemen-engine/Ikemen-GO/assets/33178487/f72e6005-c5b3-4e4a-90f3-db487ccc7709)
There are some invisible characters at the end of the def file. Remove them and it will work on nightly.

**Reply by Freshprince24** [reply]
Thanks
